modules/twinklespeedy.js

```diff
@@ -1409,7 +1409,7 @@ Twinkle.speedy.callbacks = {
 				return;
 			}
 
-			var xfd = /(?:\{\{([rsaiftcm]fd|md1|proposed deletion)[^{}]*?\}\})/i.exec( text );
+			var xfd = /(?:\{\{((?:article for deletion|template for discussion)\/dated|[cfm]fd|proposed deletion)[^{}]*?\}\})/i.exec( text );
```

Comment:
Did you test this?  AFAICS `template for discussion/dated` is insufficient because `{{subst:tfd}}` adds an `{{invoke` before hitting any closing `}}`

Reply:
I see. Just replacing that `[^{}]` with the good old `.` should work.